version.properties

```diff
@@ -2,11 +2,11 @@
 
 # WordPress version information
 wordpress.versionName=17.7-rc-1
-wordpress.versionCode=1068
+wordpress.versionCode=1071
 
-wordpress.zalpha.versionName=alpha-302
-wordpress.zalpha.versionCode=1069
+wordpress.zalpha.versionName=alpha-303
+wordpress.zalpha.versionCode=1070
```

Comment:
Note that in practice there was no alpha-303 / versionCode 1070, because for release and hotfix builds we don't build zalpha, so I bumped this for nothing when I prepared the version bumps for the hotfix, only realizing later that there would be no new zalpha.

Given zalpha.versionCode < wordpress.versionCode the gap is already there and too late to fix, so I figured it's not a big deal to commit the bump and admit we skipped/burned a versionCode there